cmake/KokkosFFT_tpls.cmake

```diff
@@ -0,0 +1,27 @@
+function(get_tpls_list tpls_list)
```

Comment:
You should add a documentation in the header to describe what this function does.